Failing github.ci run test

I'm not getting this locally, but it is occurring on all github run tests.

```
Run mkdir -p ./html/web/sites/simpletest && chmod 777 ./html/web/sites/simpletest
Running phpunit in 4 processes with /var/www/html/vendor/phpunit/phpunit/phpunit

Configuration read from /var/www/html/phpunit.xml.dist

.......................F.                                         25 / 25 (100%)

Time: 05:10.089, Memory: 10.00 MB

There was 1 failure:

1) Drupal\Tests\localgov_services_navigation\FunctionalJavascript\EntityReferenceServicesAutocompleteTest::testEntityReferenceAutocompleteWidget
Failed asserting that actual size 2 matches expected size 1.

/var/www/html/vendor/phpunit/phpunit/src/Framework/Constraint/Constraint.php:122
/var/www/html/vendor/phpunit/phpunit/src/Framework/Constraint/Constraint.php:55
/var/www/html/web/modules/contrib/localgov_services/modules/localgov_services_navigation/tests/src/FunctionalJavascript/EntityReferenceServicesAutocompleteTest.php:177
/var/www/html/vendor/phpunit/phpunit/src/Framework/TestResult.php:728

FAILURES!                               
Tests: 25, Assertions: 393, Failures: 1.
Error: Process completed with exit code 1.
```

--- a/modules/localgov_services_navigation/tests/src/FunctionalJavascript/EntityReferenceServicesAutocompleteTest.php
+++ b/modules/localgov_services_navigation/tests/src/FunctionalJavascript/EntityReferenceServicesAutocompleteTest.php
@@ -161,17 +161,18 @@ class EntityReferenceServicesAutocompleteTest extends WebDriverTestBase {
     $autocomplete_field->setValue('Page');
     $this->getSession()->getDriver()->keyDown($autocomplete_field->getXpath(), ' ');
     $assert_session->waitOnAutocomplete();
-
     $results = $page->findAll('css', '.ui-autocomplete li');
 
     $this->assertCount(2, $results);
     $assert_session->pageTextContains('Landing Page 1');
     $assert_session->pageTextContains('Landing Page 1 » Page Sub 1');
 
+    $autocomplete_field->setValue('');
+    $assert_session->waitForElementRemoved('css', '.ui-autocomplete li');
+
     $autocomplete_field->setValue('Sub');
     $this->getSession()->getDriver()->keyDown($autocomplete_field->getXpath(), ' ');
     $assert_session->waitOnAutocomplete();
-
     $results = $page->findAll('css', '.ui-autocomplete li');
 
     $this->assertCount(1, $results);

